enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/JWTUtil.java

```diff
@@ -246,5 +250,50 @@ public static AbstractAPIMgtGatewayJWTGenerator getApiMgtGatewayJWTGenerator() {
         }
         return jarFilesList;
     }
+
+    public static Map<String, JWTTransformer> loadJWTTransformers() {
+        JWTTransformer jwtTransformer = null;
+        List<String> jarFilesList = getJarFilesList();
+        Map<String, JWTTransformer> jwtTransformersMap = new HashMap<>();
+        for (int fileIndex = 0; fileIndex < jarFilesList.size(); fileIndex++) {
+            try {
+                String pathToJar = JwtConstants.DROPINS_FOLDER + jarFilesList.get(fileIndex);
```

Comment:
Is the dropins folder value hard coded or got from environment variable

Reply:
It is hard coded. It refers to `/home/wso2/lib/dropins/` location.